Players' operations

nurietzsche2021 opened the pull request:
In this file player programmer has already all the operations needed to
control players

nurietzsche2021 committed:
Players' operations
In this file player programmer has already all the operations needed to
control players

nurietzsche2021 committed:
OPERATIONS FOR UI
I already finished UI operations requested. Let me know if anything else
is needed.

nurietzsche2021 committed:
MERGE OF ALL OPERATIONS
Since I've finished all the operations and they use simillar functions,
I propose using just the operations.ex file, which includes all
necessary functions for GAME, PLAYER and UI. The rest of functions in
file are private so they don't appear in API.

ale1311sv reviewed (approved):
Nuria, those operation players functions couldn't be better and clear! Thank you for your hard work!

nurietzsche2021 committed:
FINAL CHANGES
In this last commit you can see how I offer different options to use my
operations. I'd rather use just one module (programmed in operations.ex)
but, just in case every team of project (GAME, PLAYER and UI) wants their own module I've left all the other files.
